src/openrct2-ui/windows/TextInput.cpp

```diff
@@ -172,6 +172,7 @@ class TextInputWindow final : public Window
         {
             Invalidate();
             window_set_resize(this, WW, height, WW, height);
+            height = newHeight; // need to update newHeight
```

Comment:
Comments should add something. This is more or less stating the obvious.

```suggestion
            height = newHeight;
```

@IntelOrca Is the call to `window_set_resize()` really necessary? I mean, it is called with the same values for minHeight/maxHeight and minWidth/maxWidth and with the old values to boot.

Reply:
I don't know, sorry.